src/Windows/Avalonia.Win32/ITaskBarList2VTable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using static Avalonia.Win32.Interop.UnmanagedMethods;
+
+namespace Avalonia.Win32
+{
+    delegate void MarkFullscreenWindow(IntPtr This, IntPtr hwnd, [MarshalAs(UnmanagedType.Bool)] bool fullscreen);
```

Comment:
Are these delegates/classes part of the win32 API? If so, should probably be in `UnmanagedMethods` with the rest of the win32 interop stuff?

Reply:
moved